_docs/scientists/index.md

```diff
@@ -7,4 +7,16 @@ has_toc: true
 permalink: /_docs/science
 ---
 # Intro
+Welcome to the Half Earth docs for Scientists. These docs cover specific elements of the Half-Earth Project in terms of data wrangling and management, but also some general tips to work with ESRI's `arcgis API for Python`, `arcpy` and Arcgis Online. 
+# How to add to this documentation
```

Comment:
Probably this would belong to the README of the project, but lets keep it here until we revamp the README with instructions for the whole dev cycle (running the project in local and all that stuff, you know)